content/zurihac2022/index.html

```diff
@@ -379,16 +379,14 @@ <h1>Registration</h1>
       </p>
       <ul>
         <li>
-          A breakdown of your cost estimates (flight, train tickets,
-          hotel/hostel/airbnb, food...).
+          Your total cost estimate. Send us receipts of transportation and
+          accomodation that you booked. If that's not possible, please
+          send us screenshots of the transport/accomodation that you indend to
+          book.
```

Comment:
I think we should leave this out; people may want to apply for the scholarships with an estimate, since they won't have anything booked yet. When we confirm their scholarship through email, we can state that they will need to send us receipts for everything?

Reply:
yeah, that also works, and should reduce the overhead